keps/sig-node/4265-proc-mount/README.md

```diff
@@ -0,0 +1,497 @@
+
+No, only allows a user to access the Unmasked ProcMountType
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes, though a reboot or drain will be necessary to limit any containers that originally got access to Unmasked ProcMountType.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+The API server will need to recreate the pods that had the request scrubbed of the ProcMountType field (so the original intent can be fulfilled).
+
+###### Are there any tests for feature enablement/disablement?
+
+No
+
+### Rollout, Upgrade and Rollback Planning
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+Inconsistent feature gate enablement is the only way. All supported versions of CRIs (CRI-O and containerd) have support for this feature.
+
+###### What specific metrics should inform a rollback?
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+They will be, specifically cases where the Kubelet is restarted.
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+No
+
+### Monitoring Requirements
+
+###### How can an operator determine if the feature is in use by workloads?
+
+`kubectl get pods --all-namespaces -o jsonpath="{range .items[*]}{.metadata.name}{' '}{.spec.containers[*].securityContext.procMount}{'\n'}{end}"  | grep -i unmasked`
+Will print all pods that has an Unmasked ProcMountType, along with the pod name.
+
+###### How can someone using this feature know that it is working for their instance?
+
+- [ ] Events
+  - Event Reason: 
+- [ ] API .status
+  - Condition name: 
+  - Other field: 
+- [x] Other (treat as last resort)
+  - Details: Container created with the Unmasked ProcMountType have paths [here](https://github.com/kubernetes/kubernetes/blob/964529b/pkg/securitycontext/util.go#L193) as writable, not read only.
+    - "/proc/asound",
+	- "/proc/acpi",
+	- "/proc/kcore",
+	- "/proc/keys",
+	- "/proc/latency_stats",
+	- "/proc/timer_list",
+	- "/proc/timer_stats",
+	- "/proc/sched_debug",
+	- "/proc/scsi",
+	- "/sys/firmware",
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+N/A
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+- [x] Metrics
+  - Metric name: `kubelet_pod_start_sli_duration_seconds`
+  - [Optional] Aggregation method:
+  - Components exposing the metric: kubelet
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+### Dependencies
+
+###### Does this feature depend on any specific services running in the cluster?
+
+- A CRI implementation that supports this feature
+    - All supported versions currently do.
```

Comment:
Does that include all implementations of CRI?

Reply:
of containerd+ cri-o

Reply:
update the KEP to be specific about which do.